commit-graph.c

```diff
@@ -1450,6 +1512,13 @@ static int write_commit_graph_file(struct write_commit_graph_context *ctx)
 						4 * ctx->num_extra_edges;
 		num_chunks++;
 	}
+	if (ctx->bloom) {
+		chunk_offsets[num_chunks + 1] = chunk_offsets[num_chunks] + 4 * ctx->commits.nr;
+		num_chunks++;
+
+		chunk_offsets[num_chunks + 1] = chunk_offsets[num_chunks] + 4 * 3 + ctx->total_bloom_filter_size;
```

Comment:
is `4*3` because we have 3 ui32's in the structure header ??

Reply:
@jeffhostetler Yeah! I call it out in the format doc.